Only output failing models and violated rules per default in HumanReadableFormatter (#77)
PicnicSupermarket/dbt-score: docs/index.md

In this example, the model `customers` scores the maximum value of `10.0` as it
passes all the rules. It also is awarded a golden medal because of the perfect

> score. By default a passing model without rule violations will only be shown if
> we pass the `---show_all` flag.

Reviewer: ```suggestion
score. By default a passing model with or without rule violations will not be shown, 
unless we pass the `--show-all` flag.
```

In this example, the model `customers` scores the maximum value of `10.0` as it
passes all the rules. It also is awarded a golden medal because of the perfect

> score. By default a passing model with or without rule violations will not be

Reviewer: ```suggestion
score. By default a passing model or source with or without rule violations will not be
```
